Code/Tools/ProjectManager/Source/PythonBindings.cpp

```diff
@@ -323,6 +315,19 @@ namespace O3DE::ProjectManager
 
             RedirectOutput::Intialize(PyImport_ImportModule("azlmbr_redirect"), &PythonBindings::OnStdOut, &PythonBindings::OnStdError);
 
+            // Add custom site packages after initializing the interpreter above.  Calling Py_SetPath before initialization
+            // alters the behavior of the initializer to not compute default search paths. See
+            // https://docs.python.org/3/c-api/init.html#c.Py_SetPath
+
+            AZ::IO::FixedMaxPath thirdPartyFolder = AzToolsFramework::EmbeddedPython::PythonLoader::GetDefault3rdPartyPath(false);
+
+            AZStd::vector<AZStd::string> extendedPaths;
+            AzToolsFramework::EmbeddedPython::PythonLoader::ReadPythonEggLinkPaths(thirdPartyFolder, m_enginePath.c_str(), extendedPaths);
+            if (extendedPaths.size())
```

Comment:
I think for documenting the intent of the logic, checking if the list of egg link paths is not empty should be done.
```suggestion
            if (!extendedPaths.empty())
```